apps/explorer/src/components/footer/Footer.tsx

```diff
@@ -1,61 +1,131 @@
+                    <Text variant="body/medium" color="steel-darker">
+                        Docs
+                    </Text>
+                </Link>
+            </li>
+            <li>
+                <Link variant="text" href="https://mystenlabs.com/#community">
+                    <Text variant="body/medium" color="steel-darker">
+                        GitHub
+                    </Text>
+                </Link>
+            </li>
+            <li>
+                <Link variant="text" href="https://discord.gg/sui">
+                    <SocialDiscord24 />
+                </Link>
+            </li>
+            <li>
+                <Link variant="text" href="https://twitter.com/SuiNetwork">
+                    <SocialTwitter24 />
+                </Link>
+            </li>
+            <li>
+                <Link
+                    variant="text"
+                    href="https://www.linkedin.com/company/mysten-labs/"
+                >
+                    <SocialLinkedin24 />
+                </Link>
+            </li>
+        </ul>
     );
 }
 
 function Footer() {
     return (
-        <footer className="bg-gray-75 px-5 py-10 md:px-10 md:py-14">
-            <nav className="mx-auto grid grid-cols-1 gap-8 md:mx-0 md:grid-cols-4 md:gap-10 xl:w-1/2">
-                <div className="order-last mx-auto md:order-first md:mt-0">
-                    <div className="h-full space-y-2 md:flex md:flex-col md:justify-between">
-                        <SuiLogoIcon className="mx-auto md:mx-0" />
-                        <div className="mt-auto">
-                            <Text
-                                color="white"
-                                variant="pSubtitleSmall/semibold"
-                            >
-                                &copy;
-                                {`${new Date().getFullYear()} Sui. All
-                                rights reserved.`}
-                            </Text>
-                        </div>
+        <footer className="bg-gray-40 px-5 py-10 md:px-10 md:py-14">
+            <nav className="flex flex-col gap-7.5">
+                <div className="flex w-full flex-col items-center justify-between gap-7.5 md:flex-row">
+                    <div className="h-10 text-hero-dark">
+                        <SuiWordmark />
                     </div>
+                    <FooterLinks />
                 </div>
-                <div className="col-span-1 grid grid-cols-4 md:col-span-3">
-                    <FooterLinks links={footerLinks} />
+
+                <div className="h-[1px] w-full bg-gray-45" />
+                <div className="flex w-full items-center justify-between">
+                    <div className="h-full space-y-2">
+                        <Text
+                            color="steel-darker"
+                            variant="pSubtitleSmall/medium"
+                        >
+                            &copy;
```

Comment:
is this fine to keep on the legal side of things?

Reply:
not sure what you mean here - the date? or the content of the text?

Reply:
oh sorry I meant the copyright for Sui, I wasn't sure if we were going to move the copyright to Mysten Labs or something with the ownership transfer

Reply:
this mirrors what we have on the Sui Foundation website: 
<img width="349" alt="image" src="https://user-images.githubusercontent.com/122397493/235723565-b4b42e06-7c71-4810-b613-1c78db59b41c.png">